Add subsubsection to explain how to change the path when the position… (#136)
gitextensions/GitExtensionsDoc: source/submodules.rst

|Remove        | Remove the submodule from the repository                                                                        |
+--------------+-----------------------------------------------------------------------------------------------------------------+

> Change submodule path
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Delete the existing submodule and add it again in the new location after changing the submodule location.
> 
> 

Reviewer: Suggest not adding a header (the ^^^^^ length should match the text exactly if kept) and to reword a little.
```suggestion
To change a submodule path, delete the existing submodule, move the filesystem directory and add it again in the new location.

```
